modules/core/src/main/java/org/apache/ignite/internal/processors/metastorage/persistence/DistributedMetaStorageImpl.java

```diff
@@ -406,6 +432,9 @@ private void onMetaStorageReadyForWrite(ReadWriteMetastorage metastorage) {
 
     /** {@inheritDoc} */
     @Override @Nullable public <T extends Serializable> T read(@NotNull String key) throws IgniteCheckedException {
+        if (keysToSkip.contains(key))
```

Comment:
Why do we disallow `read` operation?
All we need is skip keys that can't be unmarshalled on the node startup.

Reply:
Fixed. Explicit operations will not be ignored